x-pack/plugins/security_solution/common/test/index.ts

```diff
@@ -5,17 +5,29 @@
  * 2.0.
  */
 
-// For the source of these roles please consult the PR these were introduced https://github.com/elastic/kibana/pull/81866#issue-511165754
+import serverlessRoleDefinitions from '@kbn/es/src/serverless_resources/security_roles.json';
+import type essRoleDefinitions from './ess_roles.json';
```

Comment:
Q: Why is this imported as `type`.

Reply:
`essRoleDefinitions  is used only for type inference and the linter automatically autocorrects `import` to `import type`. TS is able to infer JSON types quite well.

Reply:
It's a bit odd is all. The json import above is imported correctly. I'm guessing it has something to do with package inference.